__tests__/src/__snapshots__/index.spec.js.snap

```diff
@@ -10,19 +10,18 @@ Object {
 }
 `;
 
-exports[`toMatchImageSnapshot should fail when diff result is unknown 2`] = `
-"Expected image to match or be a close match to snapshot.
-[1m[31mSee diff for details:[39m[22m [31mpath/to/result.png[39m"
-`;
+exports[`toMatchImageSnapshot passes diffImageToSnapshot everything it needs to create a snapshot and compare if needed 2`] = `""`;
 
 exports[`toMatchImageSnapshot should fail when snapshot has a difference beyond allowed threshold 2`] = `
-"Expected image to match or be a close match to snapshot.
+"Expected image to match or be a close match to snapshot. There is a 80% difference
```

Comment:
Looking at the message now I can see it reads awkwardly because it doesn't know when to say `a` vs. `an`. So either add logic to handle that or rephrase in a way that does not require that grammar logic.